code/game/objects/items.dm

```diff
@@ -578,7 +578,7 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 /obj/item/proc/get_dismemberment_chance(obj/item/bodypart/affecting)
 	if(affecting.can_dismember(src))
-		if((sharpness || damtype == BURN) && w_class >= 3)
+		if((sharpness || damtype == BURN || (damtype == BRUTE && (affecting.owner.has_trait(TRAIT_EASYDISMEMBER)))) && w_class >= 3)

```

Comment:
i don't know how this will play out, but remember that other species have this trait as well

Reply:
you should add a species call back here to just allow dismember multiples and move all this code into the species classes

Reply:
All current usages `EASYDISMEMBER` also have `LIMBATTACHMENT`. So, yes, Skeletons and Zombies would start having their limbs knocked off by suitably large blunt objects (3 = `WEIGHT_CLASS_NORMAL`) but it's still easy for them to pop them back on. 

Balance wise de-limbing zombies with a toolbox is a problem, I guess?

Reply:
Yes, they can't put it back their arms back on because of the claw

Reply:
Fuck. I forgot about the claws.